From 70f985d5515081968598f5e58f4b5aa3e3b994f6 Mon Sep 17 00:00:00 2001
From: Thierry Carrez <thierry@openstack.org>
Date: Tue, 13 Dec 2011 17:10:44 +0100
Subject: [PATCH] Fix issue in test_s3_malicious_tarballs

The previous commit backported a master commit which added a test
that cannot run successfully on stable/diablo due to a missing
import.

Change-Id: I6e5a1142e1902f4fc658ed376c91e9535379c12f
---
 nova/tests/image/test_s3.py |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/nova/tests/image/test_s3.py b/nova/tests/image/test_s3.py
index 4ab36a0..c22e348 100644
--- a/nova/tests/image/test_s3.py
+++ b/nova/tests/image/test_s3.py
@@ -18,6 +18,7 @@
 import os
 
 from nova import context
+from nova import exception
 from nova import test
 from nova.image import s3
 
-- 
1.7.6.5

